scalability_and_performance/index.adoc

```diff
@@ -0,0 +1,61 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="index"]
+= {product-title} scalability and performance overview
+include::_attributes/common-attributes.adoc[]
+:context: index
+
+// ifndef::openshift-dedicated,openshift-rosa[]
+{product-title} offers best practices and utilities to help you optimize the performance and scale of your {product-title} clusters. The following documentation provides information on recommended performance and scalability practices, reference design specifications, optimization and low latency tuning.
+
+To engage with Red Hat support, see xref:../support/getting-support.adoc#getting-support[Getting support].
+
+// endif::openshift-dedicated,openshift-rosa[]
+
+[NOTE]
+====
+Some performance and scalability Operators have release cycles that are independent from {product-title} release cycles. For more information see link:access.redhat.com/support/policy/updates/openshift_operators[Openshift Operators].
+====
+
+[discrete]
+==== Recommended performance and scalability practices
+
+xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-control-plane-practices.adoc#recommended-control-plane-practices[Recommended control plane practices]
+
+xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-infrastructure-practices.adoc#recommended-infrastructure-practices[Recommended infrastructure practices]
+
+xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc#recommended-etcd-practices[Recommended etcd practices]
+
+[discrete]
+==== Telco Reference design specifications
+
+xref:../scalability_and_performance/telco_ref_design_specs/ran/telco-ran-ref-design-spec.adoc#[Telco RAN DU specification]
+
+xref:../scalability_and_performance/telco_ref_design_specs/core/telco-core-rds-overview.adoc#[Telco core reference design specification]
+
+[discrete]
+==== Planning, Optimization and Measurement
```

Comment:
```suggestion
==== Planning, optimization, and measurement
```
ISG loves serial commas and sentence case.

Reply:
Roger that. done!